Mathlib/Topology/Order/ScottTopology.lean

```diff
@@ -210,8 +210,8 @@ lemma UpperSetTopology_Monotone_coinduced_LE {t₁ : TopologicalSpace α} [Topol
   rw [IsScott.topology_eq β]
   apply UpperSetTopology_Monotone_coinduced_LE' hf
 
-lemma Monotone_From_UpperSetTopology_Continuous {_ : TopologicalSpace α} [Topology.IsUpperSet α]
-    {_ : TopologicalSpace β} [IsScott β] {f : α → β} (hf : Monotone f)  : Continuous f :=
+lemma Monotone_From_UpperSetTopology_Continuous [TopologicalSpace α] [Topology.IsUpperSet α]
```

Comment:
This kind of lemmas makes me think you need typeclasses of the form "upper sets are open in the topology" and "open sets are upper sets in the topology".
```suggestion
lemma continuous_of_monotone_of_upperSetTopology [TopologicalSpace α] [Topology.IsUpperSet α]
```

Reply:
@YaelDillies more generally:
```
import Mathlib.Topology.Order.Lattice

class IsLE (α : Type*) (t₀ : TopologicalSpace α) [t : TopologicalSpace α] [Preorder α] : Prop where
  topology_le_Topology : t ≤ t₀

class IsGE (α : Type*) (t₀ : TopologicalSpace α) [t : TopologicalSpace α] [Preorder α] : Prop where
  topology_ge_Topology : t ≥ t₀
```
?

Reply:
I don't think that `Topology.IsLE` is a great idea. I would rather have specialized version for the cases we care about.

Reply:
Perhaps I should remove these lemmas until someone has a use for them?

Reply:
@YaelDillies I've removed these morphisms, as I'm not convinced that they're actually useful.